ext/hash/hash.c

```diff
@@ -980,8 +985,9 @@ PHP_FUNCTION(hash_pbkdf2)
 	zend_bool raw_output = 0;
 	const php_hash_ops *ops;
 	void *context;
+	HashTable *args;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "Sssl|lb", &algo, &pass, &pass_len, &salt, &salt_len, &iterations, &length, &raw_output) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "Sssl|lbh!", &algo, &pass, &pass_len, &salt, &salt_len, &iterations, &length, &raw_output, &args) == FAILURE) {
```

Comment:
```suggestion
	if (zend_parse_parameters(ZEND_NUM_ARGS(), "Sssl|lbh", &algo, &pass, &pass_len, &salt, &salt_len, &iterations, &length, &raw_output, &args) == FAILURE) {
```
In line with non-nullable parameter in signature.

Reply:
Actually, now that I look again, was it intentional to change this function? The hash_pbkdf2() change is not in the stub file and not tested either.